src/function/scalar/string/levenshtein.cpp

```diff
@@ -0,0 +1,98 @@
+#include "duckdb/function/scalar/string_functions.hpp"
+#include "duckdb/common/vector_operations/vector_operations.hpp"
+#include "duckdb/common/string_util.hpp"
+
+#include <ctype.h>
+#include <algorithm>
+
+namespace duckdb {
+
+static idx_t Minimum(const idx_t &x, const idx_t &y, const idx_t &z) {
+	return std::min(x, std::min(y, z));
```

Comment:
Please avoid using generic names like `Minimum` for functions. In DuckDB we support amalgamation builds which means all source files will be combined into a single `.cpp` file. Generic names are a potentially source of conflicts, since `Minimum` might be used somewhere else in the codebase. 

Since the function is only used in one place, I recommend inlining the function.

Also please use `MinValue` instead of std::min (from [duckdb/common/helper.hpp](https://github.com/cwida/duckdb/blob/master/src/include/duckdb/common/helper.hpp)). `std::min` has bitten us before on certain platforms.

Reply:
Noted.

Reply:
Used MinValue and simplified.